x-pack/legacy/plugins/spaces/server/routes/api/v1/example.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { InternalRouteDeps } from '.';
+
+export function initExampleApi(deps: InternalRouteDeps) {
+  const { http, savedObjects, routePreCheckLicenseFn } = deps;
+
+  http.route({
+    method: 'GET',
+    path: '/api/spaces/v1/without_wrapper',
+    async handler(request: any) {
+      const { getScopedSavedObjectsClient } = savedObjects;
+
+      const savedObjectsClient = getScopedSavedObjectsClient(request, {
+        excludedWrappers: ['spaces'],
```

Comment:
Sorry, I haven't worked with SavedObjectClient so far. Why we need this functionality?

Reply:
There are very limited use-cases for excluding wrappers. 
We are trying to use this for the [Copy to Space](https://github.com/elastic/kibana/issues/37286) feature. Specifically, we will be introducing an API endpoint within the Spaces plugin which needs to be able to reach across spaces. By removing the spaces wrapper from that specific endpoint's instance, we can control which space (or, `namespace`) the SavedObjectsClient is targeting for each request.